pkg/commands/builder/status.go

```diff
@@ -134,6 +134,23 @@ func printBuilderReadyStatus(bldr *v1alpha2.Builder, writer io.Writer) error {
 		return err
 	}
 
+	cpTableWriter, err := commands.NewTableWriter(writer, "BuildpackName", "     BuildpackKind")
```

Comment:
nit: the column headers should be more readable `Buildpack Name`, `Buildpack Kind`

Reply:
Done